src/features/script-load.js

```diff
@@ -18,26 +18,36 @@ systemJSPrototype.register = function (deps, declare) {
 
 systemJSPrototype.instantiate = function (url, firstParentUrl) {
   const loader = this;
-  return new Promise(function (resolve, reject) {
-    const script = document.createElement('script');
-    script.charset = 'utf-8';
-    script.async = true;
-    script.crossOrigin = 'anonymous';
-    script.addEventListener('error', function () {
-      reject(new Error('Error loading ' + url + (firstParentUrl ? ' from ' + firstParentUrl : '')));
+  if (url.endsWith('.json')) {
+    return fetch(url).then(function (resp) {
+      return resp.json();
```

Comment:
A JSON parse error would throw here - effectively resulting in an instantiation error. I believe the specification intention is that a JSON parser error should in fact be an execution error.

This can be done by returning the response `text()`, and then separating the `JSON.parse` call to be contained within the `execute()` function.

It would also be good to include a test for a JSON parse error.